osquery/tables/yara/yara.cpp

```diff
@@ -39,71 +40,221 @@ FLAG(bool,
      true,
      "Call malloc_trim() after YARA scans (linux)");
 #endif
+
 FLAG(uint32,
      yara_delay,
      50,
      "Time in ms to sleep after scan of each file (default 50) to reduce "
      "memory spikes");
 
+HIDDEN_FLAG(bool,
```

Comment:
I'll have to go back through the design discussion to refresh my understanding. But I think it's fine to not have this flag at all and allow sigrule to be enabled all the time. I might also recommend `sigrule` be changed to a HIDDEN flag to prevent logging the rule in results.

Reply:
I added the flag after the initial discussion about data exfiltration and disabled it by default. However, it also gets addressed by disabling the string table and making the rule strings private if they are passed using `sigrule`. 

I will remove the flag and make the `sigrule` column hidden.